cmd/compgen/README.md

```diff
@@ -15,6 +15,14 @@ types are currently supported:
 - [ ] [Processors](https://opentelemetry.io/docs/collector/configuration/#processors)
 - [ ] [Exporters](https://opentelemetry.io/docs/collector/configuration/#exporters)
 
+## Installing `compgen`
+
+From within `cmd/compgen` run `go install .`. This will make build the `compgen`
```

Comment:
We should have a ticket to fix the name of this so that it doesn't conflict with zsh.

Reply:
https://github.com/orgs/liatrio/projects/25/views/1?pane=issue&itemId=76028102